test/end-to-end/edit_journal/edit_journal.spec.js

```diff
@@ -0,0 +1,146 @@
+/* global browser, element, by */
+const chai = require('chai');
+const helpers = require('../shared/helpers');
+const FU = require('../shared/FormUtils');
+const GU = require('../shared/GridUtils');
+const components = require('../shared/components');
+
+helpers.configure(chai);
+const expect = chai.expect;
+
+describe('Edit Posting Journal', () => {
+  const path = '#!/journal';
+  const gridId = 'journal-grid';
+  before(() => helpers.navigate(path));
+
+  var accountKey = 11;
+
+  it('edits a transaction change an account', function () {
+    // click the "grouping" button
+    FU.buttons.grouping();
+    element.all(by.css('[class="ui-grid-icon-plus-squared"]')).get(0).click();
+    element.all(by.css('[class="fa fa-edit"]')).get(0).click();
+    const accountNumberCell = GU.getCellName(gridId, 1, 4);
+    // simulate dbl click
+    browser.actions().mouseMove(accountNumberCell).doubleClick().perform();
+    
+    accountNumberCell.element(by.css("input")).sendKeys(protractor.Key.chord(protractor.Key.CONTROL, "a"));
+    accountNumberCell.element(by.css("input")).sendKeys(protractor.Key.BACK_SPACE);
+
+    accountNumberCell.element(by.css("input")).sendKeys(accountKey);
+    element.all(by.css('[title="1100 - Test Capital One"]')).click();
+
+    element.all(by.css('[class="fa fa-save"]')).click();
```

Comment:
These tests will break if we change our icon font.  Could you put a `data-` tag on this element and use it as a selector?  Something like `[data-method="save"]`.